### DISCUSSION TITLE
I never see the animation of cutting off trophies (with a knife), although I set this option in the mod settings to 100.

### DISCUSSION THREAD

**Discussion by discohouse57** [Aelto/W3_RandomEncounters_Tweaks#192] [Q&A]
I tried to set other values - the same thing.

**Comment by Aelto** [comment]
The trophy animation only plays if the monsters around you dropped trophies. It could be you have trophies disabled?

**Reply by discohouse57** [reply]
Here is a screenshot - everything seems to be enabled:
![Screenshot 2023-04-12 165423](https://user-images.githubusercontent.com/41947290/231479901-d7cb4934-1cec-435b-bf79-042cc5292d12.jpg)
